### ISSUE TITLE
Please change link colour

### ISSUE THREAD

**Issue by goetzk** [omeka/omeka-s-enduser#52]
Hi,
I just got as far as creating a fork of the documentation to try and add a link , only to discover there was already one in the same paragraph I was working on.
Is it possible to change the link colour so its more readily visible?

thanks,

**Comment by zerocrates** [comment]
Your issue is the red color? It's fairly distinctive... and also generally it's the color S itself uses. The "visited" links do maybe inch toward the the too-subtle though.

**Comment by goetzk** [comment]
Hi @zerocrates ,

I've looked at work and home (both Macs) and still can't tell the difference, I guess the red in question falls in the area red/green colourblind hits.

![image](https://user-images.githubusercontent.com/8732490/53001534-fa37df00-347e-11e9-8c11-e8b0e43bab47.png)

PS.
I swear I included links in my post :/ Let me try again.

I'm talking about the user-manual frontpage section:
https://omeka.org/s/docs/user-manual/frontpage/

**Comment by mebrett** [comment]
It looks like it's dichromatic - 
![screen shot 2019-02-19 at 9 23 30 am](https://user-images.githubusercontent.com/2314070/53021844-13fa0780-3428-11e9-8d23-9d5f294ad8fa.png)
Screencap from [Coblis](https://www.color-blindness.com/coblis-color-blindness-simulator/) with an image of our documentation.

**Comment by zerocrates** [comment]
To my knowledge red is particularly problematic in this area... we'd be talking about going to a completely different color (or possibly adding other decoration like underlines?)

@kimisgold

**Comment by goetzk** [comment]
Hi @zerocrates ,
I find the links on Utas' instance ( #e32213 ) easier to see than the documentation ( #A91919 ) but I don't know how applicable that information is to other themes.

https://exhibit.utas.edu.au/s/olive-pink/page/welcome

**Comment by mebrett** [comment]
@kimisgold Could we change the styling so that links are both differently colored and underlined? (which is what the [WCAG Guidelines](https://www.w3.org/TR/WCAG21/#animation-from-interactions) use)

**Comment by kimisgold** [comment]
I swapped the colors of the links' default and hover since the original hover has more brightness and saturation. I also added dotted borders that solidify on hover. How do we feel about these now?

**Comment by mebrett** [comment]
It looks really good to me. I checked it with a colorblind simulator, and between the contrast and the borders it was always apparent what the links where.
<img width="732" alt="Screen Shot 2020-06-09 at 4 39 34 PM" src="https://user-images.githubusercontent.com/2314070/84197500-067aed80-aa70-11ea-8f93-edb9a869ad5a.png">

**Comment by goetzk** [comment]
Looking much better, thanks!
